### ISSUE TITLE
After completion of the ms.prod value update from the Graph API docs, we still have a number of files, that have not picked the ms.prod value.

### ISSUE THREAD

**Issue by msewaweru** [MicrosoftDocs/microsoftgraph-docs-powershell#214]
For example: [Get-MgApplicationLogo](https://github.com/MicrosoftDocs/microsoftgraph-docs-powershell/blob/main/microsoftgraph/graph-powershell-1.0/Microsoft.Graph.Applications/Get-MgApplicationLogo.md)

**Comment by timayabi2020** [comment]
@msewaweru This is noted. Issue is on our open api files which are missing the external docs parameter for  the related cmdlets. So its not easy to fetch the ms-prod value. However, I am following up with the metadata guys.

**Comment by timayabi2020** [comment]
@msewaweru please review this issue with the latest merge.

**Comment by msewaweru** [comment]
@timayabi2020 - issue persists. See [here](https://github.com/MicrosoftDocs/microsoftgraph-docs-powershell/blob/main/microsoftgraph/graph-powershell-1.0/Microsoft.Graph.Applications/Get-MgApplicationLogo.md).

**Comment by CelesteDG** [comment]
Hi @timayabi2020 - FYI that ms.prod and ms.technology are being retired and we're switching to ms.service and ms.subservice. The metadata updates is ongoing right now for Microsoft Graph API. We've requested similar subservice values but ms.service will be Microsoft Graph PowerShell.

@FaithOmbongi to confirm when the PowerShell metadata has been added. I don't see the new ms.service and ms.subservice values added to the allowlist just yet (allowlist here: https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main)

FYI @csmulligan @omondiatieno
